specs/electra/beacon-chain.md

```diff
@@ -1025,7 +1028,8 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
     if not self.notify_new_payload(
             execution_payload,
             parent_beacon_block_root,
-            execution_requests_list):
+            execution_requests_list,
+            target_blobs_per_block):
```

Comment:
`execution_requests_list` and `target_blobs_per_block` should also be passed to the `is_valid_block_hash` call as they are part of the block hash check